public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -197,3 +199,20 @@ export const getStyles = (theme: GrafanaTheme2) => ({
     },
   }),
 });
+
+function shouldDisableInstallation(plugin: CatalogPlugin) {
+  if (
+    plugin.type === PluginType.renderer ||
+    plugin.type === PluginType.secretsmanager ||
+    (plugin.isEnterprise && !featureEnabled('enterprise.plugins')) ||
+    !plugin.isPublished ||
+    plugin.isCore ||
+    plugin.isDisabled ||
```

Comment:
I think we also need this but @andresmgot can confirm.
```suggestion
    plugin.isDisabled ||
    plugin.isPreinstalled.withVersion ||
```

Reply:
I missed this, yes, `isPreinstalled.withVersion` also needs to be checked. I see the latest code includes this so LGTM!